docs/guide/wallets.md

```diff
@@ -79,6 +79,13 @@ For the invoice to work you must have a publicly accessible URL in your LNbits.
 - `OPENNODE_API_ENDPOINT`: https://api.opennode.com/
 - `OPENNODE_KEY`: opennodeAdminApiKey
 
+### GetAlby
+
+For the invoice to work you must have a publicly accessible URL in your LNbits. No manual webhook setting is necessary.
+
+- `LNBITS_BACKEND_WALLET_CLASS`: **GetAlbyWallet**
+- `GETALBY_API_ENDPOINT`: https://api.getalby.com/
+- `GETALBY_API_KEY`: getAlbyApiKey
```

Comment:
would it be good to note how the user can generate this API key? which I believe is a personal access token here, right?

https://getalby.com/developer/access_tokens/new

So I would probably call it `ALBY_ACCESS_TOKEN`

Reply:
Will add it to the docs section